site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -283,29 +276,14 @@ export const WorkspaceScheduleForm: FC<WorkspaceScheduleFormProps> = ({
   )
 }
 
-export const ttlShutdownAt = (now: dayjs.Dayjs, workspace: Workspace, tz: string, formTTL: number): string => {
-  // a manual shutdown has a deadline of '"0001-01-01T00:00:00Z"'
-  // SEE: #1834
-  const deadline = dayjs(workspace.latest_build.deadline).utc()
-  const hasDeadline = deadline.year() > 1
-  const ttl = workspace.ttl_ms ? workspace.ttl_ms / (1000 * 60 * 60) : 0
-  const delta = formTTL - ttl
-
-  if (delta === 0 || !isWorkspaceOn(workspace)) {
-    return Language.ttlHelperText
-  } else if (formTTL === 0) {
+export const ttlShutdownAt = (formTTL: number): string => {
+  if (formTTL < 1) {
+    // Passing an empty value for TTL in the form results in a number that is not zero but less than 1.
     return Language.ttlCausesNoShutdownHelperText
   } else {
-    const newDeadline = dayjs(hasDeadline ? deadline : now).add(delta, "hours")
-    if (newDeadline.isSameOrBefore(now)) {
-      return `⚠️ ${Language.ttlCausesShutdownHelperText} ${Language.ttlCausesShutdownImmediately} ⚠️`
-    } else if (newDeadline.isSameOrBefore(now.add(30, "minutes"))) {
-      return `⚠️ ${Language.ttlCausesShutdownHelperText} ${Language.ttlCausesShutdownSoon} ⚠️`
-    } else {
-      return `${Language.ttlCausesShutdownHelperText} ${Language.ttlCausesShutdownAt} ${newDeadline
-        .tz(tz)
-        .format("MMM D, YYYY h:mm A")}.`
-    }
+    return `${Language.ttlCausesShutdownHelperText} ${dayjs.duration(formTTL, "hours").humanize()} ${
+      Language.ttlCausesShutdownAfterStart
+    }.`
```

Comment:
*review:* As we no longer can depend on the workspace deadline, I'm just simplifying this a whole bunch and showing the humanized duration.